### DISCUSSION TITLE
See "Viewed" files within File Tree Navigation Pane.

### DISCUSSION THREAD

**Discussion by Austinito** [community/community#18285] [Pull Requests]
It would be nice to see what files I have already viewed within the File Tree Navigation Pane.

This way I can navigate quickly to the _NOT_ viewed files from my File Tree Navigation Pane. Currently, I open an already viewed file, and it will open (from collapsed) and I won't notice I've already viewed it until looking into top-right corner.

Could we get a marker for the ones that I have seen?

![Viewed files Screenshot](https://user-images.githubusercontent.com/19643883/173115190-a4dd91c3-a979-495e-9167-d554669050e0.png)

**Comment by LuigiCardamone** [comment]
I agree, it would be awesome!
An extra feature could be a switch to hide all "viewed" files from the File Tree.
